src/test/scala/rise/elevate/tiling.scala

```diff
@@ -34,8 +34,7 @@ class tiling extends test_util.Tests {
   def betaEtaEquals(a: Rise, b: Rise): Boolean = {
     val na = BENF(a).get
     val nb = BENF(b).get
-    val uab: Rise = toBeTyped(na) !: nb.t
-    makeClosed(uab) == makeClosed(nb)
+    makeClosed(toBeTyped(na)) =~= makeClosed(nb)
```

Comment:
is there a reason to remove the type annotation `!: nb.t`?

Reply:
It makes constraint solving fail, but I don't yet know why

Reply:
So it appears to be something related to the type annotation making some implicit variables explicit, I need to investigate more

Reply:
Fixed! Eta reduction wasn't operating correctly: it relies on the comparison of identifier names, without taking into account their types.